Reflection/Reflector.php

```diff
@@ -21,9 +21,6 @@ public static function export();
     /**
      * Returns the string representation of any Reflection object.
      *
-     * Please note that since PHP 8.0 this method is absent in this interface
```

Comment:
Could you please leave this part as is? It's correct and clarifies the existence of the method.

Reply:
@MaXal it's not correct